NextcloudTalk.xcodeproj/project.pbxproj

```diff
@@ -2590,8 +2594,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/nextcloud/ios-communication-library";
 			requirement = {
-				kind = exactVersion;
-				version = 0.98.0;
+				branch = develop;
+				kind = branch;
```

Comment:
Or tag a new version, as there seems to be some improvements after 99.7....

Reply:
Yes, you are right :) I already talked with @marinofaggiana and the idea is to make a release in ios-communication-lib which include unified search.

Reply:
> Do we really want to point to a development branch here? This will break sooner or later. I'd prefer targeting/releasing 99.7 with unified search: [nextcloud/ios-communication-library#117](https://github.com/nextcloud/ios-communication-library/pull/117)

When ready, will be the 99.8, and not before.